### PR TITLE
documentation: rare disease subset curation

### PR DESCRIPTION
**Pull request by sabrinatoro**

### CODE REVIEW

File: docs/editors-guide/rare-disease-curator.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Rare Disease Curation
+
+Rare diseases in Mondo are created based on 3 main subsets. Rules for editing these subsets depends on the subset type. Before editing rare disease information in Mondo, please familiarize yourself with how rare diseases are represented in Mondo [here](https://mondo.readthedocs.io/en/latest/editors-guide/rare-disease-subset/).
+
+It is important to remember that the rare disease subsets are updated automatically at the time of the monthly Mondo release. It is, therefore, crucial that editing rare disease information is done by following the instructions below for these edits to be retained after the release.
+
```
Comment (sabrinatoro) [inline docs/editors-guide/rare-disease-curator.md:6]: @matentzn is that true? Or should curators run something like 
sh run.sh make mondo.owl MIR=false IMP=false
?

File: docs/editors-guide/rare-disease-curator.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+**1. Rare disease from authority sources.**
+These terms have an `in_subset` annotation `X_rare`  (where X is the rare disease authority).
+These subsets are maintained automatically:
+- a list of rare diseases is maintained for each source here:
+    - src/ontology/subsets/nord-subset.template.tsv
+    - src/ontology/subsets/gard-subset.template.tsv
+- during the Mondo release, the `X_rare` subset is updated based on the Mondo terms on the authoritative source list
+
+Until we have set a process with the sources to maintain this list automatically, Mondo editors should update this list of rare diseases manually **upon request by the authoritative source**.
```
Comment (sabrinatoro) [inline docs/editors-guide/rare-disease-curator.md:15]: @matentzn is that right?

File: docs/editors-guide/rare-disease-curator.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+1. Create or open a GitHub issue reported by the authoritative source (ie NORD, GARD) to add or remove a term from the authoritative rare disease list.   
+1. Open the rare disease list (from src/ontology/subsets) in a text editor:
+     1. To add a new term, add a new line with the Mondo ID to add to the rare subset. Please make sure that the line is similar to the previous one: this document is a robot template, and therefore the information in each column is important.
+     1. To remove a term from the list, delete the line
+1. The term will be added to the rare disease list once the PR is merged, but the subset annotation will not be added until the monthly Mondo release has run.
+
+Note: rare diseases from Orphanet are maintained by a completely automated process. If an update is needed, please create a GitHub issue for the technical team.
```
Comment (sabrinatoro) [inline docs/editors-guide/rare-disease-curator.md:22]: @matentzn is that right?

File: docs/editors-guide/rare-disease-curator.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+**3. Inferred rare disease.**
+These terms have an `in_subset` annotation `inferred_rare`. This subset is created automatically and refers to the ontological children of rare disease terms.
+This subset should **NOT** be manually curated.
+
+
+#### Note about rare disease subset during term obsoletion/merging.
+
+1. If a MONDO term (MONDO id) is obsoleted, the Mondo term will not appear in the rare disease subset, regardless of whether the disease is considered rare by one of the authoritative rare disease sources.
+This removal of an obsolete term from the rare disease subset is automated. Curators should confirm that the removal from the rare disease subset by reviewing the Mondo release following the term obsoletion. Any error should be reported to the technical team via a GitHub issue.
```
Comment (sabrinatoro) [inline docs/editors-guide/rare-disease-curator.md:44]: @matentzn Do we need to update the rare subsets manually when we obsolete a term, or is it done automatically?

File: docs/editors-guide/rare-disease-curator.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+2. If an authoritative rare source obsoletes a disease term or removes a disease from their rare disease list, the corresponding Mondo term will lose its corresponding "X_rare" subset annotation for this particular source.
+This removal from an authoritative source subset will be automated. However, until the pipeline is fully set and functional, curators will update the authority rare disease subset list manual (see "Rare disease from authority sources" section above)
+
+3. If a MONDO term (Mondo-1) is merged with another term (Mondo-2), its rare subset annotations will be transferred to this term (Mondo-2).
+Curators merging a rare disease term should ensure that the Mondo ID is merged or added to the authoritative source list. (see "Rare disease from authority sources: section above).
```
Comment (sabrinatoro) [inline docs/editors-guide/rare-disease-curator.md:50]: @matentzn what is happening when merging terms? Do we need to update the rare subsets manually (ie replacing the Mondo-1 by Mondo-2)?

File: docs/editors-guide/rare-disease-curator.md (hunk 1 above)
Comment (matentzn) [reply docs/editors-guide/rare-disease-curator.md:6]: The sentence here is correct. A curator that really wants to rebuild the rare subset (the owl file) can run:

```
sh run.sh make rare_subset
```

File: docs/editors-guide/rare-disease-curator.md (hunk 2 above)
Comment (matentzn) [reply docs/editors-guide/rare-disease-curator.md:15]: Yes, right now:

NORD and GARD are managed here: https://github.com/monarch-initiative/mondo/tree/master/src/ontology/subsets

Orphanet is managed in mondo-edit.obo

File: docs/editors-guide/rare-disease-curator.md (hunk 3 above)
Comment (matentzn) [reply docs/editors-guide/rare-disease-curator.md:22]: The subset is automatically generated yes. But the mappings to Orphanet are of course curated.

File: docs/editors-guide/rare-disease-curator.md (hunk 4 above)
Comment (matentzn) [reply docs/editors-guide/rare-disease-curator.md:44]: The authoritative_rare subsets are maintained as they are, but the "rare" subset excludes all deprecated classes.

File: docs/editors-guide/rare-disease-curator.md (hunk 5 above)
Comment (matentzn) [reply docs/editors-guide/rare-disease-curator.md:50]: Never update any rare subsets manually other than mondo_rare..

**Comment by sabrinatoro** [comment]
@matentzn I updated the document based on our discussion. 

This PR is ready for review and merge

File: docs/editors-guide/rare-disease-curator.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+These subsets are maintained automatically:
+- a list of rare diseases is maintained for each source here:
+    - src/ontology/subsets/nord-subset.template.tsv
+    - src/ontology/subsets/gard-subset.template.tsv
+- during the Mondo release, the `X_rare` subset is updated based on the Mondo terms on the authoritative source list
+
+Until we have set a process with the sources to maintain this list automatically, Mondo editors should update this list of rare diseases manually **upon request by the authoritative source**.
+1. Create or open a GitHub issue reported by the authoritative source (ie NORD, GARD) to add or remove a term from the authoritative rare disease list.   
+1. Open the rare disease list (from src/ontology/subsets) in a text editor:
+     1. To add a new term, add a new line with the Mondo ID to add to the rare subset. Please make sure that the line is similar to the previous one: this document is a robot template, and therefore the information in each column is important.
```
Comment (nicolevasilevsky) [inline docs/editors-guide/rare-disease-curator.md:18]:
```suggestion
     1. To add a new term, add a new line with the Mondo ID to add to the rare subset. Please make sure that the line is similar to the previous one: this document is a ROBOT template, and therefore the information in each column is important.
```

**Review by nicolevasilevsky — APPROVED** [review]
looks good!

**Pull request merged**